src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -34,29 +34,40 @@ export type ExportAnnotationsProps = {
 type ExportFormat = {
   /** Unique format identifier used also as file extension */
   value: 'json' | 'csv' | 'txt' | 'html';
-  name: string;
+  /** The title to be displayed in the listbox item */
+  title: string;
+
+  /**
+   * The title to be displayed in the Select button.
+   * Falls back to `name` when not provided.
+   */
+  shortTitle?: string;
+
   description: string;
 };
 
 const exportFormats: ExportFormat[] = [
   {
     value: 'json',
-    name: 'JSON',
+    title: 'JSON',
     description: 'For import into another Hypothesis group or document',
   },
   {
     value: 'txt',
-    name: 'Text',
+    title: 'Plain text (TXT)',
+    shortTitle: 'Text',
```

Comment:
IMO with the improved titles the description could be omitted for plain text / rich text. I think it is still useful for JSON / CSV though.

Reply:
I agree, but I think it would look a bit "unbalanced", so I would personally keep them.